fpmsyncd/routesync.cpp

```diff
@@ -550,17 +551,20 @@ void RouteSync::onEvpnRouteMsg(struct nlmsghdr *h, int len)
     FieldValueTuple intf("ifname", intf_list);
     FieldValueTuple vni("vni_label", vni_list);
     FieldValueTuple mac("router_mac", mac_list);
+    FieldValueTuple proto("protocol", proto_str);
```

Comment:
as i see, this is only for evpn routes and not for regular routes, right? and no impact to warmboot etc

Reply:
That's correct. The flow will happen only for EVPN. No impact to warmboot.